### PR TITLE
charm-tools 2.7.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Moisan** [comment]
```
pkg_resources.DistributionNotFound: The 'pathspec<=0.3.4' distribution was not found and is required by charm-tools
```
Also `brew audit` is failing.

**Comment by chenrui333** [comment]
Should be fixed now, tested locally.

File: Formula/charm-tools.rb [hunk 1]
```diff
@@ -24,29 +24,37 @@ class CharmTools < Formula
     sha256 "caabb9c22961a3413ac85cd1e5525ec9ca80daeba6555f4f60802b6c256e252b"
   end
 
+  # pyyaml>=3.11,<4.3
```
Comment (SMillerDev) [inline Formula/charm-tools.rb]: What's with the comments?

Comment (Moisan) [reply Formula/charm-tools.rb]: We usually use the latest version available on pypi unless the package forces a specific version or range of versions. If we don't use the latest version, we typically add a comment pointing the package restrictions.

Comment (SMillerDev) [reply Formula/charm-tools.rb]: Can/have we documented that somewhere?

Comment (chenrui333) [reply Formula/charm-tools.rb]: that is just my note (grabbed from setup.py, https://github.com/juju/charm-tools/blob/master/setup.py#L47)

**Pull request closed without merging**